pkg/controllers/csi/metadata/models.go

```diff
@@ -14,20 +14,20 @@ type TimeStampedModel struct {
 
 // TenantConfig holds info about a given configuration for a tenant.
 type TenantConfig struct {
-	CodeModule CodeModule `gorm:"foreignKey:Version;references:DowloadedCodeModuleVersion"`
+	CodeModule CodeModule `gorm:"foreignKey:DowloadedCodeModuleVersion;references:Version"`
 	TimeStampedModel
-	UID                        string `gorm:"primaryKey" json:"UID,omitempty"`
-	Name                       string `gorm:"not null"`
-	DowloadedCodeModuleVersion string
-	ConfigDirPath              string `gorm:"not null"`
-	TenantUUID                 string
+	UID                        string  `gorm:"primaryKey" json:"UID,omitempty"`
+	Name                       string  `gorm:"not null"`
+	DowloadedCodeModuleVersion string  `gorm:"not null"`
```

Comment:
we have a typo here: DowNloaded... fix also needs to be in CodeModule association comment further up.

Reply:
fixed in 43eb3821